### DISCUSSION TITLE
What is the equivalent of 1000 bonus point spin achieved in stable for lazer?

### DISCUSSION THREAD

**Discussion by Rhythmic-Ocean** [ppy/osu#25528] [Q&A] [answered]
I found a map which, when played by auto on lazer barely gets a tick mark, but when played on stable gets 1000 bonus points. And since lazer uses score V2 I was curious what is the equivalent of 1000 bonus point spin achieved in stable for lazer (is it just that tick mark?) as this concerns the RC as well. I don't wanna repeatedly go the stable just to check if the spinner in my map can achieve 1000 bonus points or not to fulfill the rule of RC.
Map I am talking about: https://osu.ppy.sh/beatmapsets/1988612#osu/4283591
Spinner I'm talking about: 00:19:813 (1) -   of last diff of the map.

On stable:
https://imgur.com/cAIOUyy

On lazer:
https://imgur.com/a/yUpoj5t

That's a ranked map btw.

Also, is there no 'update the beatmap' button like in stable that u usually get after making some changes into someone else's beatmap? I tried making the spinner longer to see the bonus score in lazer of the map above and I didn't get any 'update the beatmap' button. So, I deleted the beatmap and re-downloaded it. Something like that on lazer would be convenient, I think. 

Thanks.

P.S: Also that spinner is note (1) on stable but note (7) on lazer. Aren't spinners supposed to automatically be note (1)? Weird ig. 

P.S 1: I also have another problem, like if I downloaded the replay of the map I just played right at the score board that appears right after I finish playing the map, it downloads just fine. After exiting the score board for the first time, I can still download the replay but it's empty (0 KB). A short clip of that: 

https://imgur.com/a/itF7nAW

**Comment by bdach** [comment, marked as answer]
> What is the equivalent of 1000 bonus point spin achieved in stable for lazer?

50 points per bonus spin. See https://github.com/ppy/osu/wiki/Scoring#hit-results.

> Map I am talking about: https://osu.ppy.sh/beatmapsets/1988612#osu/4283591
> Spinner I'm talking about: 00:19:813 (1) - of last diff of the map.

Spinners in lazer have been rebalanced to be easier to complete, so it is possible some of these rebalances changed the number of achievable bonus spins for this spinner to 0 as it is rather short.

> Also, is there no 'update the beatmap' button like in stable that u usually get after making some changes into someone else's beatmap?

https://github.com/ppy/osu/issues/23524

> P.S: Also that spinner is note (1) on stable but note (7) on lazer. Aren't spinners supposed to automatically be note (1)? Weird ig.

We may fix this but it is cosmetic at best and does not realistically affect anything since the spinner still forces a new combo on the next non-spinner object.

Dunno about the download issue.

**Reply by Rhythmic-Ocean** [reply]
Thanks
